src/CPE/cpelang.c

```diff
@@ -125,11 +114,6 @@ char *cpe_lang_model_detect_version_priv(xmlTextReader *reader)
 	return oscap_strdup("2.3");
 }
 
-char * cpe_lang_model_detect_version(const char* file)
-{
-	return cpe_lang_model_detect_version_priv(NULL);
```

Comment:
why not also remove the function above, is anything using it?

Reply:
cpe_lang_model_detect_version_priv is used only at src/source/oscap_source.c:387. It could be replaced by its code. On the other hand, it is a private function. I would prefer to define this string as a constant in some of the header files.